bedrock/firefox/templates/firefox/challenge-the-default/landing-base.html

```diff
@@ -4,37 +4,19 @@
 file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-{% extends "base-protocol.html" %}
+{% extends "firefox/base/base-protocol.html" %}
 
 {% from "macros-protocol.html" import picto with context %}
 
 {% block page_title %}{{ ftl('firefox-home-firefox-protect-your') }}{% endblock %}
-
-{% block page_title_suffix %}{{ ftl('firefox-home-mozilla') }}{% endblock %}
-
-{% block page_desc %}
-  {{ seo_desc }}
-{% endblock %}
-{% block page_image %}
-  {% if LANG == "fr" %}
-    {{ static('img/firefox/challenge-the-default/ctd-share-fr.png') }}
-  {% else %}
-    {{ static('img/firefox/challenge-the-default/ctd-share.png') }}
-  {% endif %}
-{% endblock %}
-{% block page_favicon %}{{ static('img/favicons/firefox/favicon.ico') }}{% endblock %}
-{% block page_favicon_large %}{{ static('img/favicons/firefox/favicon-196x196.png') }}{% endblock %}
-{% block page_ios_icon %}{{ static('img/favicons/firefox/apple-touch-icon.png') }}{% endblock %}
+{% block page_title_suffix %} — {{ ftl('firefox-home-mozilla') }}{% endblock %}
+{% block page_desc %}{{ seo_desc }}{% endblock %}
```

Comment:
We should bring the `page_image` over for CTD social share images.

Changing the template did make the favicon updates unnecessary though 🙂 

```suggestion
{% block page_desc %}{{ seo_desc }}{% endblock %}
{% block page_image %}
  {% if LANG == "fr" %}
    {{ static('img/firefox/challenge-the-default/ctd-share-fr.png') }}
  {% else %}
    {{ static('img/firefox/challenge-the-default/ctd-share.png') }}
  {% endif %}
{% endblock %}
```

Reply:
Oh no worries, it started out as a simple a11y +runaway tag fix, but while testing it and after validating further it turned out there's more to tackle — maybe if I submitted the individual issues separately, It would have been easier to review in smaller focused chunks, instead of this seemingly unrelated lump of tiny changes…

Reply:
@stephaniehobson So this poster image thing is actually sorted out, just elsewhere: 

https://github.com/mozilla/bedrock/blob/7dfe7422b8d50d5945d675fbff89f629c88cab81/bedrock/firefox/templates/firefox/challenge-the-default/langs/landing.fr.html#L7-L9

Every language sets it. It only needed removing this hardcoded logic in base to start working for all;)

Reply:
All the three links I've chosen for testing above _(de, pl, fr)_ should each render its own poster, as was originally intended in [#14306/files](https://github.com/mozilla/bedrock/pull/14306/files#diff-793b0f2cc1d76024da6a9f7cf4f28fae809fb1c43d1f24c7c9b85902a289523e) additions.

NB: DE uses English branding, other languages have logos and such localized.